docs/LICENSE.md

```diff
@@ -41,21 +41,13 @@ The GNU Affero GPL version 3 has also been brought into the fold. The original A
-GPLv3 uses a new term, “convey,” and provides a definition for that term. “Convey” has the same meaning we intended for “distribute,” but now that this is explained directly in the license, it should be easy for people everywhere to understand what we meant. There are other minor changes throughout the license that will also help ensure it is applied consistently worldwide.
-When the Rules Are Broken: A Smooth Path to Compliance
-Under GPLv2, if you violated the license in any way, your rights were automatically and permanently lost. The only way to get them back was to petition the copyright holder. While a strong defense against violations is valuable, this policy could cause a lot of headache when someone accidentally ran afoul of the rules. Asking all the copyright holders for a formal restoration of the license could be burdensome and costly: a typical GNU/Linux distribution draws upon the work of thousands.
 
 GPLv3 offers a reprieve for good behavior: if you violate the license, you’ll get your rights back once you stop the violation, unless a copyright holder contacts you within 60 days. After you receive such a notice, you can have your rights fully restored if you’re a first-time violator and correct the violation within 30 days. Otherwise, you can work out the issue on a case-by-case basis with the copyright holders who contacted you, and your rights will be restored afterward.
 Compliance with the GPL has always been the top priority of the FSF Compliance Lab and other groups enforcing the license worldwide. These changes ensure that compliance remains the top priority for enforcers, and gives violators incentive to comply.
 The Latest and Greatest
 Some of these changes probably seem less important to you than others. That’s okay. Every project is different, and needs different things from its license. But odds are that a number of these improvements will help you and your work.
 And taken as a whole, all these upgrades represent something more: we made a better copyleft. It does more to protect users’ freedom, but it also enables more cooperation in the free software community. But updating the license is only part of the job: in order for people to get the benefits it offers, developers need to use GPLv3 for their projects, too. By releasing your own software under the new license, everyone who deals with it—users, other developers, distributors, even lawyers—will benefit. We hope you’ll use GPLv3 for your next release.
+<<<<<<< HEAD
```

Comment:
Looks like a merge conflict happened with the license file? These <<<<<Head and ====== lines denote that a decisions for fixing the conflict should have happened.